code/modules/power/supermatter/supermatter.dm

```diff
@@ -1176,6 +1176,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		if(prob(5))
 			zap_str -= (zap_str/10)
 			zap_count += 1
+			targets_hit = targets_hit.Copy() //Pass by ref begone
```

Comment:
Is this meant to be under this prob(5) ?

Reply:
Yeah, cause it's the only place the bolts will split, and so the only place it needs to be copied. I'll add a seperate check for that so things are cleaner. It can, I'm a dumbass, fixing now.

Reply:
I don't understand how is this supposed to work. Is a split zap meant to target same objects again or not ?

Reply:
If a zap splits, each "side" of the bolt shouldn't be using the same list of things hit.
Like if side A hits a grounding rod, side B should still be allowed to hit it

Reply:
Then shouldn't this be in inside the loop in case zap_count > 2. And copied twice in energy_ball.dm ?

Reply:
ah hell you're right, I'll update it